modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -206,16 +212,10 @@ runModule.run.meta.analysis <- function(settings) {
 ##' @return result of p<distn>(point, parama, paramb)
 ##' @export p.point.in.prior
 ##' @author David LeBauer
-p.point.in.prior <- function(point, prior){
-  prior.median <- do.call(paste('q', prior$distn, sep = ""),
+p.point.in.prior <- function(point, prior) {
+  # Why is this (below) called, and then never used?
+  prior.median <- do.call(paste0("q", prior$distn),
                           list(0.5, prior$parama, prior$paramb))
-  p.point <- do.call(paste('p', prior$distn, sep = ""),
-                     list(point, prior$parama, prior$paramb))
-  return(p.point)
-}
-#==================================================================================================#
-
-
-####################################################################################################
-### EOF.  End of R script file.      				
-####################################################################################################
+  do.call(paste0("p", prior$distn), 
+          list(point, prior$parama, prior$paramb))
```

Comment:
Explicit returns generally make for more readable code, even if not required by R.